lib/deprecation-helpers.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const { makeBadgeData } = require('./badge-data');
+
+const deprecatedServices = {
+  'gittip': new Date('2017-12-29'),
+  'gratipay': new Date('2017-12-29'),
+  'gemnasium': new Date('2018-05-15'),
+  'snap': new Date('2018-01-23'),
+  'snap-ci': new Date('2018-01-23'),
+  'cauditor': new Date('2018-02-15'),
+};
+
+const isDeprecated = function (service, now=new Date(), depServices=deprecatedServices) {
+  if (!(service in depServices)) {
+    return false;
+  }
+  if (now.getTime() >= depServices[service].getTime()) {
+    return true;
+  }
+  return false;
```

Comment:
You could `return now.getTime() >= depServices[service].getTime()` which is already a boolean.